app/templates/components/forms/wizard/basic-details-step.hbs

```diff
@@ -118,7 +118,7 @@
     <label>{{t 'Links & Social Media'}}</label>
     {{widgets/forms/link-input
       hasLinkName=true
-      fixedName=true
+      fixedName=false
```

Comment:
@niranjan94 I have made changes as you suggested. But wanted to clear out we need to disabled this field? Because with the above piece I checked it is not disabling even when `fixedName=true` and if never want to disable then I guess we can remove the field.